### ISSUE TITLE
Extract artist name from post (and not URL)

### ISSUE THREAD

**Issue by BigBlueHat** [dogpixels/patreon-helper#16]
The extension runs fine on the personal feed page, but puts everything into `_unknown`. However, each post may have a different artist (in that context) and so pulling the artist name from the post content would help organize things better.

**Comment by AnimatedEightball** [comment]
Yeah, it's also a bit weird that the folder is titled "_unknown" to begin with, it implies it was intended to show something else, but fails and puts it all in that folder.
